python/lsst/ip/isr/isrTask.py

```diff
@@ -1325,6 +1330,12 @@ def run(self, ccdExposure, camera=None, bias=None, linearizer=None,
             self.log.info("Converting exposure to floating point values.")
             ccdExposure = self.convertIntToFloat(ccdExposure)
 
+        if self.config.doBias and self.config.doBiasBeforeOverscan:
+            self.log.info("Applying bias correction.")
+            isrFunctions.biasCorrection(ccdExposure.getMaskedImage(), bias.getMaskedImage(),
```

Comment:
We decided not to add protocol numbers for now, so I thought we were relying on the sizes of the biases to detect whether the master biases were built with `isr.doOverscan=False isr.doAssembleCcd=False isr.doBiasBeforeOverscan=True`.   If that's right, wouldn't `config.doTrimToMatchCalib` defeat the check?  Maybe validate should check that it's not set with `isr.doBiasBeforeOverscan`

Reply:
We are not relying on the sizes to detect the master bias construction.  The configuration files for `constructBias.py` and `runIsr.py/processCcd.py` have to be consistent for now.  I've added the check in `validate`, because although we don't do automatic detection, those two config options should be mutually exclusive.

Reply:
Will it fail if the sizes don't match?  If so, maybe we could have a more helpful error message

Reply:
The `isrFunctions.biasCorrection` raises with a message if the bounding boxes do not match:
```
raise RuntimeError("maskedImage bbox %s != biasMaskedImage bbox %s" %
                           (maskedImage.getBBox(afwImage.LOCAL), biasMaskedImage.getBBox(afwImage.LOCAL)))

```